Update GitHub workflow example

mhitza opened the pull request:
The working directory is missing from your workflow file in this repository but stuck around in the README

With that working directory directive, molecule will fail with:

```
ERROR: 'molecule/*/molecule.yml' glob failed.  Exiting.
```

mhitza committed:
Update GitHub workflow example

The working directory is missing from your workflow file in this repository but stuck around in the README

With that working directory directive, molecule will fail with:

```
ERROR: 'molecule/*/molecule.yml' glob failed.  Exiting.
```

geerlingguy commented on README.md:
```diff
@@ -128,7 +128,7 @@ Here's the whole workflow we're targeting:
 ---
 name: CI
-'on':
+on:
```
Can you leave the `'on'` for now? `yamllint` actually identifies that as an ambiguous identifier (indeed, some YAML parsers have trouble with it and identify it as a bool instead of a string).

mhitza committed:
Keep quotes around GitHub Workflow "on" directive

As some YAML linters seem to have an issue with it https://github.com/geerlingguy/molecule-playbook-testing/pull/2#discussion_r426726887

geerlingguy commented:
Thanks!
